packages/components/src/components/data-table/_data-table-expandable.scss

```diff
@@ -63,9 +63,11 @@
   }
 
   tr.#{$prefix}--parent-row.#{$prefix}--expandable-row + tr[data-child-row] td {
-    padding-bottom: rem(23px);
     transition: padding $duration--moderate-01 motion(standard, productive);
```

Comment:
Do we want to have transition effects on the padding change? It is making it seem like the text is flying in from the left when the cell is expanded